src/components/Todos.tsx

```diff
@@ -0,0 +1,231 @@
+              aria-expanded="false"
+            >
+              Sort By
+            </button>
+            <div className="dropdown-menu" aria-labelledby="dropdownMenuButton">
+              <button
+                className={`dropdown-item ${sortBy === "name" ? "active" : ""}`}
+                onClick={() => handleSort("name")}
+              >
+                Name
+              </button>
+              <button
+                className={`dropdown-item ${sortBy === "date" ? "active" : ""}`}
+                onClick={() => handleSort("date")}
+              >
+                Date
+              </button>
+            </div>
+          </div>
+          <div>
+            {/* <span onClick={() => handleSort("asc")}>
+              <ArrowUp />
+            </span>
+            <span onClick={() => handleSort("desc")}>
+              <ArrowDown />
+            </span> */}
+              <span onClick={() => handleSort(sortBy)}>
+              {sortOrder === "asc" ? <ArrowUp /> : <ArrowDown />}
+            </span>
+          </div>
+          <div className="dropdown ms-3">
+            <button
+              className="btn btn-warning dropdown-toggle"
+              type="button"
+              id="dropdownMenuButton"
+              data-toggle="dropdown"
+              aria-haspopup="true"
+              aria-expanded="false"
+            >
+              Status
+            </button>
+            <div className="dropdown-menu" aria-labelledby="dropdownMenuButton">
+              <button
+                className="dropdown-item"
+                value="all"
+                onClick={handleStatus}
+              >
+                All
+              </button>
+              <button
+                className="dropdown-item"
+                value="complete"
+                onClick={handleStatus}
+              >
+                Completed
+              </button>
+              <button
+                className="dropdown-item"
+                value="incomplete"
+                onClick={handleStatus}
+              >
+                Incomplete
+              </button>
+            </div>
+          </div>
+        </div>
+        <div className="todos d-flex justify-content-left ms-0">
+          <table className="table ">
+            <thead>
+              <tr>
+                <th>Sr.No</th>
+                <th>Title</th>
+                <th>Date</th>
+                <th>Action</th>
+              </tr>
+            </thead>
+            <tbody>
+              {filteredData
+                .filter((item) => {
+                  return search.toLocaleUpperCase() === ""
+                    ? item
+                    : item.title.toLocaleLowerCase().includes(search);
+                })
+                .map((item: Todo, index: number) => (
+                  <tr key={item.id}>
+                    <td>{index + 1}</td>
+                    <td>
+                      <div onClick={() => navigate("/viewtodos/" + item.id)}>
```

Comment:
Write a function as navigateTodo and use here

Reply:
Done